pos_event_sale/models/pos_order.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Camptocamp SA - Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
```

Comment:
Do we have to handle the refund somewhere in that file ? I guess that refunding a pos.order should cancel the event.registration ? Don't you think ? 

(if not in the scope of your work, maybe add something in the roadmap)

Reply:
Added to roadmap